.github/workflows/validate-changelog-content-infra-tests.yml

```diff
@@ -0,0 +1,50 @@
+name: Run content infrastracture tests
+on: pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.10'
+
+      - name:  Get commit hash
+        id: get-commit
+        run: echo commit=$(git rev-parse HEAD) >> $GITHUB_OUTPUT
+
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          repository: demisto/content
+          path: content
+
+
+      - name: Setup Poetry
+        uses: Gr1N/setup-poetry@v8
+
+      - name: Install demisto-SDK from branch
+        run: |
+          cd content
+          poetry install --with ci
+          echo "uninstalling content-master version of demisto-sdk"
+          pip uninstall -y demisto-sdk
+          echo "uninstalled content-master demisto-sdk"
```

Comment:
These 2 echos are the same correct me if I'm wrong.
If so, let's remove the second one (after the command you run)
And change the first one to: `Uninstalling demisto-sdk from content master`, I think it is clearer